sdk/core/azure-core/inc/azure/core/exception.hpp

```diff
@@ -90,7 +90,7 @@ namespace Azure { namespace Core {
      */
     explicit RequestFailedException(
         const std::string& what,
-        std::unique_ptr<Azure::Core::Http::RawResponse>& rawResponse);
```

Comment:
You mentioned this was a recent change. Was that a change, one that is technically breaking?

Reply:
Yes, it is technically breaking, if the customer was creating instances of RequestFailedException with the other constructor.